profiles/monitoring/alerts/loki/bitte-loki.nix

```diff
@@ -0,0 +1,26 @@
+{ ... }: {
+  groups = [
+    {
+      concurrency = 2;
+      interval = "30s";
+      name = "bitte-loki";
+      rules = [
+        {
+          alert = "HighFail2banRate";
```

Comment:
Do we really care about this one?

Reply:
No this was an initial loki alert testing example.  We can swap it out for something realistically useful shortly.